tests/test_ping.py

```diff
@@ -8,4 +8,4 @@ def test_ping(test_app):
     
     # Then
     assert response.status_code == 200
-    
\ No newline at end of file
+    assert response.json() == {"ping": "pong", "environment": "test", "testing": True, "database_url": "postgres://root:password@db:5432/test"}
```

Comment:
what if we change our .env settings?

Reply:
RC: We could split it into more files.
@czakuou This will be not the best idea. We should inject ENV in setUp